examples/dummy_fuzz_test.cc

```diff
@@ -0,0 +1,24 @@
+/*
```

Comment:
Maybe call this file `empty_fuzz_test`? This way, it will be clear from the name what it does.